cli/src/commands/sidekick/mod.rs

```diff
@@ -45,9 +45,9 @@ pub enum SubCommand {
 	/// Get the CLI version
 	GetCliVersion(get_cli_version::Opts),
 	/// Get a public namespace token
-	GetPublicNamespaceToken(get_pub_namespace_token::Opts),
+	GetNamespacePubToken(get_namespace_pub_token::Opts),
 	/// Get a development namespace token
-	GetDevelopmentNamespaceToken(get_dev_namespace_token::Opts),
+	GetNamespaceDevToken(get_namespace_dev_token::Opts),
```

Comment:
nit: cli command is technically `Public` and `Development` and the shorthand is an alias. i personally like to use the full thing in the enum for clarify and so that's what it shows in the docs.